### DISCUSSION TITLE
Ability to add header and footer to b-dropdown

### DISCUSSION THREAD

**Discussion by balane** [bootstrap-vue/bootstrap-vue#7186] [Ideas]
I need to add a footer to a b-dropdown. Something similar to what Kendo has implemented(https://www.telerik.com/kendo-vue-ui/components/dropdowns/multiselect/custom-rendering/#toc-headers-and-footers). Seems like it would be fairly simple to add slots for these but beyond that, does anyone have a way to do it with just CSS?

**Comment by Hiws** [comment]
Looks fairly simple.

Something like:
```html
<b-dropdown text="Dropdown">
  <b-dropdown-text>Header</b-dropdown-text>
  <li style="max-height: 100px; overflow-y: auto;">
    <ul style="list-style: none; padding-inline-start: 0">
      <b-dropdown-item v-for="i in 10">Item {{ i }}</b-dropdown-item>
    </ul>
  </li>
  <b-dropdown-text>Footer</b-dropdown-text>
</b-dropdown>
```
